src/Services/UploadImageService.php

```diff
@@ -223,14 +235,14 @@ protected function setUploadedImage(string $imageAttributeName)
         }
 
         // An image is already uploaded, a new one is uploaded
-        if (ends_with($value, '.jpg') && !empty($originalValue)) {
+        if (str_contains($path_extension, $this->allowed_extensions) && !empty($originalValue)) {
```

Comment:
Why `str_contains` ? and not `in_array`?

Reply:
$path_extension can have a "?v=5665..." parameter. So it'll fail if we use in_array. Plus, str_contains is a cool laravel helper :sunglasses:

Reply:
In this case, prefers `starts_with` (because current code handle `toto.exe?v=jpg`)

Reply:
Thanks :)